db/models/articles-model.js

```diff
@@ -1,5 +1,14 @@
 const db = require("../connection");
 
+const readArticles = () => {
+  return db.query(`SELECT * FROM articles ORDER BY created_at DESC;`).then(({ rows }) => {
+    rows.forEach((article) => {
+      delete article.body;
```

Comment:
The SQL query can allow us to select which columns we want, so you shouldn't have to delete article.body here when you refactor it. 👍